src/NoteMouseHandler/NoteMouseHandler.js

```diff
@@ -0,0 +1,97 @@
+function findMap(arr, func) {
+  for (const item of arr) {
+    const result = func(item)
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->